.github/scripts/end2end/configs/zenko.yaml

```diff
@@ -52,7 +52,7 @@ spec:
         size: 20Gi
         storageClassName: "standard"
   kafkaCleaner:
-    interval: 30s
+    interval: 5m
```

Comment:
5m will make the kafka-cleaner test muuuchhh longer...
is KFKC period really an issue, now that you fixed everything else?
do we have an idea on the mecanism by which it breaks the re-install tests? (e.g. cpu measurement if load-related, race-condition in the pra, ...)

Reply:
This fixes the re-install part, before that change, the second install took way too much time...
I don't know exactly why, but passing it to 5mn fixed the reinstall...
I can probably shorten it to 2/3 minutes but 30s is way too fast i guess...

Reply:
Made it back to 30s and it works